src/algorithms/detect_cycle.c

```diff
@@ -13,7 +13,7 @@
 bool IsAcyclicGraph(const QueryGraph *qg) {
 	assert(qg);
 
-	bool cycle = false; // Return value.
+	bool acycle = true;
```

Comment:
I think `acyclic` is a more obvious variable name.

Reply:
👍